cerbero/config.py

```diff
@@ -234,6 +246,7 @@ def get_env(self, prefix, libdir, py_prefix):
         monopath = self._join_path (os.path.join(libdir, 'mono', '4.5'),
             os.path.join(libdir, 'mono', '4.5', 'Facades'))
 
+        variants = ':'.join(self.variants.get_enabled())
```

Comment:
I think All variants should be listed in their current state, no just enabled ones.